cli/resolver.rs

```diff
@@ -320,7 +320,12 @@ impl NpmModuleLoader {
 
     let code = if self.cjs_resolutions.contains(specifier) {
       // translate cjs to esm if it's cjs and inject node globals
-      let code = String::from_utf8(code)?;
+      let code = match String::from_utf8_lossy(&code) {
+        Cow::Owned(code) => code,
+        // SAFETY: `String::from_utf8_lossy` guarantees that the result is valid
+        // UTF-8 if `Cow::Borrowed` is returned.
+        Cow::Borrowed(_) => unsafe { String::from_utf8_unchecked(code) },
+      };
```

Comment:
It's surprising there isn't a version of this on `String`.

Reply:
Yeah... I have to reimplement this helper frequently 🥲

Maybe I should make a PR to the standard library.